composer.json

```diff
@@ -11,10 +11,10 @@
   ],
   "homepage": "https://github.com/alleyinteractive/wp-big-pit",
   "require": {
-    "php": "^8.1",
+    "php": "^8.2",
     "alleyinteractive/composer-wordpress-autoloader": "^1.0",
-    "alleyinteractive/wp-psr16": "^0.1.0",
-    "alleyinteractive/wp-type-extensions": "^2.1",
+    "alleyinteractive/wp-psr16": "dev-feature/clock-update",
```

Comment:
This will be updated to the latest tagged release after https://github.com/alleyinteractive/wp-psr16/pull/4.